sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3524,14 +3524,14 @@ _ZN4sycl3_V17handler20associateWithHandlerEPNS0_6detail30UnsampledImageAccessorB
 _ZN4sycl3_V17handler20memcpyToDeviceGlobalEPKvS3_bmm
 _ZN4sycl3_V17handler20setKernelCacheConfigENS1_23StableKernelCacheConfigE
 _ZN4sycl3_V17handler20setStateSpecConstSetEv
+_ZN4sycl3_V17handler21setKernelWorkGroupMemEm
```

Comment:
There seem to be lots of re-ordering in the file which are unrelated. I would suggest to outline them into a separate `[NFC]` PR to simplify code review here.